app.py

```diff
@@ -53,22 +55,29 @@ def main():
                     _LOGGER.info("repository %s is archived, therefore skipped", repo.full_name)
                 else:
                     repos.add(repo.full_name)
-        except GithubException:
+        except UnknownObjectException:
             _LOGGER.error("organization %s was not recognized by GitHub API", org)
 
-    repos_raw = repositories.split(",") if repositories is not None else []
+    repos_raw = list_data(repositories)
     for repo in repos_raw:
         try:
             if gh.get_repo(repo).archived:
                 _LOGGER.info("repository %s is archived, therefore skipped", repo.full_name)
             else:
                 repos.add(repo)
-        except GithubException:
+        except UnknownObjectException:
             _LOGGER.error("Repository %s was not recognized by GitHub API", repo)
 
     schedule_repositories(repositories=repos)
 
 
+def list_data(str_list: str) -> List[str]:
```

Comment:
all this method could be created in thoth-common as for https://github.com/thoth-station/common/blob/4152294f4410a943959e9e4e8f903e01d33ee029/thoth/common/openshift.py#L781

Reply:
I've issued this in #26 so that now we can have this here and later (after POC) can move it to thoth.common. Sounds reasonable @pacospace ?